src/lib/keccak-old.unit-test.ts

```diff
@@ -0,0 +1,272 @@
+import { test, Random } from './testing/property.js';
```

Comment:
should we remove this? whats the benefit of keeping both this and `keccak.unit-test.ts`? Can we combine the two?

Reply:
on it